### ISSUE TITLE
Invalid SemVar format

### ISSUE THREAD

**Issue by BennetLeff** [Calrama/llvm-d#11]
I'm getting the following build error whenever I run `dub` or `dub build`:
```
Error executing command build:
Invalid SemVer format: LLVM_3_6_2
```

My dub.json file:

```
{
	"name": "project",
	"description": "A minimal D application.",
	"copyright": "Copyright © 2016, bennetleff",
	"authors": ["bennetleff"],
	"dependencies": {
        "llvm-d": "~>1.0.0",
    },
    "version": "LLVM_3_6_2",
}
```

Am I specifying the LLVM version wrong?

**Comment by Calrama** [comment]
Yes, your [dub config](https://code.dlang.org/package-format?lang=json) is invalid. You need to do this:

```json
{
    "name": "project",
    "description": "A minimal D application.",
    "copyright": "Copyright © 2016, bennetleff",
    "authors": ["bennetleff"],
    "dependencies": {
        "llvm-d": "~>1.0.0",
    },
    "versions": [ "LLVM_3_6_2" ],
}
```

Dub complains about the fact that you tried to use a [*D version*](https://dlang.org/spec/version.html) (which is a compile time identifier passed into D code) as the value for a key that requires a [*Semantic version*](https://code.dlang.org/package-format?lang=json#version-specs) (which is a package management thing that D code never sees). However, even if you had used a correct semantic version there, it would've been invalid, since *version* makes no sense outside of a dependency declaration.

In short:
- *version* is used to set the semantic version for a surrounding dependency declaration.
- *versions* is used to set compile time D versions passed by dub to the compiler.

Compare:
```json
{
    "name": "project",
    "description": "A minimal D application.",
    "copyright": "Copyright © 2016, bennetleff",
    "authors": ["bennetleff"],
    "dependencies": {
        "llvm-d": {
            "version": "~>1.0.0"
        }
    },
    "versions": [ "LLVM_3_6_2" ],
}
```

**Comment by BennetLeff** [comment]
Alright that makes sense.You might consider changing the README from calling it the "version" field to the "versions" field.

**Comment by Calrama** [comment]
Done, sorry about that.

**Comment by BennetLeff** [comment]
Thanks
